src/sass/components/_contacts.scss

```diff
@@ -0,0 +1,136 @@
+.contacts{
+    background-color: $pageBg;
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items:center;
+    padding:86px 50px 113px;
+    
+    
+
+
+&__head{
+    display:none;
+}
+    
+&__box{ 
+    display:flex;
+    flex-direction: column;
+    gap:20px;
+    max-width:480px;
+    width:100%;
+    justify-content: center;
+   
+
+}
+    
+
+
+&__container{
+    background-color:$white;
+    box-shadow:0px 8px 30px $contactsBoxShadow;
+    border-radius:24px;
+    padding:43px 20px;
+    
+}
+  
+
+
+&__title{
+@include font-mixin(inherit, 500, 12px, 1.86, 0.04em, $white);
+border-radius:6px;
+text-align: center;
+padding:2px 25px;
+
+
+    
+}
+&__title--cafe{
+background-color: $CoffeeBG;;
+}
+
+&__title--foodtruck{
+background-color: $lime;}
+
+&__city{
+@include font-mixin(inherit, 500, 12px, 1.86, 0.04em, $body);
+padding:25px 0 10px;
+display:block;
+
+}
+&__description{
+@include font-mixin(inherit, 500, 12px, 1.85, unset, $text);
+padding-bottom:25px;
+border-bottom: 1px solid $gray;
+}
+
+&__opening-hours{
+padding:25px 0;
+gap:25px;
+display:flex;
+flex-direction: column;
+}
+
+&__weekdays{
+@include font-mixin(inherit, 500, 12px, 1.86, unset, $body);
+padding-bottom:5px;
+}
+
+&__hours{
+@include font-mixin(inherit, 500, 12px, 1.86, unset, $brand-pink);
+}
+
+&__details{
+border-top: 1px solid $gray;
+padding-top:25px;
+}
+
+
+&__info{
+@include font-mixin(inherit, 500, 12px, 1.5,unset, $text);
+}
+
+&__buttons{
+display:flex;
+flex-direction: column;
+justify-content: center;
+padding-top:54px;
+gap:10px;
+
+
+}
+
+
+@media(min-width:768px){
```

Comment:
media nie zagnieżdżasz w bloku, dajesz to osobno pod wszystkimi stylami. Czyli w sumie tylko zamknij blok wcześniej i tyle